src/__tests__/ParseFile-test.js

```diff
@@ -67,13 +67,21 @@ describe('ParseFile', () => {
   });
 
   it('can extract data type from base64', () => {
-    const file = new ParseFile('parse.txt', {
+    const file = new ParseFile('parse.png', {
```

Comment:
Not sure whether this was deliberately a `txt` extension to demonstrate that the image type is derived from the data, not from the extension?

Reply:
It doesn't matter that much. The type is never taken from the extension. In other tests the extension corresponds to the specified type.